index.js

```diff
@@ -0,0 +1,58 @@
+function validatePassword(password) {
+  return (password) &&
+    (isValidLength(password)) &&
+    (isLowerCase(password)) &&
+    (isUpperCase(password)) &&
+    (isNumber(password)) &&
+    (isSpecial(password))
+
+}
+
+function isValidLength(password) {
+  if (password.length < 8) return false
+  return true
+}
+
+function testLetter(character, lower, upper) {
```

Comment:
This is a bit of "overloading of terms" as you're using lower and upper to define the range but upper and lower already mean case in the code. Using start and end would be better.